src/game/Player.cpp

```diff
@@ -2447,14 +2447,12 @@ bool Player::IsGroupVisiblefor (Player* p) const
 
 bool Player::IsInSameGroupWith(Player const* p) const

```

Comment:
The Problem is Visibility.GroupMode = 1 as set in the server config is not working for invisibilty, seems to only use IsInSameGroupWith for Visibilty.